docs/CUSTOMIZING.md

```diff
@@ -0,0 +1,66 @@
+# Customizing guide
+
+This document describes how you can create your own app for an event.
+
+## 1. Required information
+
+The following information are needed to configure the app for your event.
+
+- Schedule URL which provides Frab compatible XML
+- Session URL template, e.g. `https://awesome-event.com/2021/events/%1$s.html`
+- Server backend type, one of: `pentabarf`, `frab`, `pretalx`
+- Schedule feedback URL template (optional), e.g. `https://awesome-event.com/2021/events/%s/feedback/new`
+- Google Play URL, e.g. `https://play.google.com/store/apps/details?id=com.awesome.event.schedule`
+- Event URL, e.g. `https://awesome-event.com/2021`
+- Start and end date of the event
+- Email address for bug reports
+- Engelsystem URL (optional), e.g. `https://engelsystem.de/awesome-event/shifts-json-export?key=YOUR_KEY`
+
+## 2. Required graphic assets and colors
+
+The following graphics and colors are needed in order to customize the look and feel of the app.
+
+- Launcher icon (SVG)
+- Notification icon (SVG)
+- Dialog logo (SVG)
+- Text and background colors (HEX)
+- Tracks background colors (HEX)
+- Bell icon (PNG)
+- Video recording icons (PNG)
+
+## 3. Your custom app step by step
+
+In some of the steps it is the easiest to copy and adapt configuration settings, folders and files from an existing flavor.
+
+1. Fork the repository
+2. Create a new branch for your event, e.g. `awesome-event-2021`
+3. Add a new product flavor in *app/build.gradle* e.g. `awesome2021` and the corresponding folder e.g. `app/src/awesome2021`
+4. Configure all required properties in your flavor (`applicationId`, `versionName`, `buildConfigField`, `resValue`)
+5. Enable showing the app disclaimer via `SHOW_APP_DISCLAIMER` to acknowledge its origin
+6. Add a new signing config in *app/gradle.properties*
+7. Customize texts for the languages which you want to offer (`values/strings.xml`, `values-de/strings.xml`, ...)
+8. Add track resource names in *res/xml/track_resource_names.xml*
+9. Customize track colors in *res/values/colors_congress.xml*
+10. Customize app colors in *res/values/colors.xml*
+11. Add a launcher icons in different resolutions as *res/mipmap-[...]/ic_launcher.png*
+12. Add a notification icon in different resolutions as *res/drawable-[...]/ic_notification.png*
```

Comment:
Either "Add a \w icon" or "Add \w icons", but not a mix of both, for number 11 and 12.

Reply:
:heavy_check_mark: Fixed in f49d549452dbe6b2e10b8050ace391d0ee12b55f.